packages/blade/src/components/Icons/SearchIcon/SearchIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import { Svg, Path } from '../_Svg';
```

Comment:
Search icon looks weird. Is this intended? Also couldn't find it on Figma. Can you point me to the correct 

<img width="58" alt="Screenshot 2022-08-18 at 3 21 10 PM" src="https://user-images.githubusercontent.com/24487274/185366509-53987bc2-c2ef-48d2-8980-7db1b25a14dc.png">

This is how it should be:
<img width="77" alt="Screenshot 2022-08-18 at 3 24 15 PM" src="https://user-images.githubusercontent.com/24487274/185367197-74006e5b-2021-4f51-88fb-3f757fe0fb7f.png">

Reply:
Resolved now